mapfishapp/src/main/webapp/app/js/GEOR_tools.js

```diff
@@ -529,6 +529,22 @@ GEOR.tools = (function() {
                     return (ids.indexOf(r.id) > -1);
                 }), true);
             }
+        },
+
+        /**
+         * APIMethod: getAddonsState
+         * Retrieve informations about addons
+         */
+        getAddonsState: function() {
+            return previousState;
```

Comment:
We should rename the `previousState` var into `state` in this file...
It would better reflect reality IIRC.